packages/templates/nextjs/.env.example

```
@@ -1,4 +1,6 @@
 DODO_PAYMENTS_API_KEY=your-api-key
 DODO_PAYMENTS_WEBHOOK_KEY=your-webhook-secret
 DODO_PAYMENTS_RETURN_URL=https://yourdomain.com/checkout/success
-DODO_PAYMENTS_ENVIRONMENT="test"or"live"
\ No newline at end of file
+DODO_PAYMENTS_ENVIRONMENT="test"or"live"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)